### ISSUE TITLE
Artikkel - Understanding map projections

### ISSUE THREAD

**Issue by ellvtr** [GeoForum/veiledning05#2]
Denne artikkel er veldig gøy og lettlest og har et par kule illustrasjoner:
http://visualoop.com/blog/28606/understanding-map-projections

Se også: http://sandbox.azavea.com/projection-overlays/

![map-projections-as-human-heads](https://cloud.githubusercontent.com/assets/12038749/7854752/a76e3d5e-051c-11e5-8387-aeef8e6e232f.jpg)

![projections](https://cloud.githubusercontent.com/assets/12038749/7854790/e2709316-051c-11e5-993b-3602a02bd00b.png)

**Comment by spohner** [comment]
Ja, den var veldig bra. Illustrasjonene beskriver temaet godt! Skal se på hvordan jeg kan få inn noe av det.
